tests/dataset_runner.py

```diff
@@ -99,7 +99,7 @@ def run(self, dataset_fixture, run_name_prefix="test"):
                 self.wait_for_primary_bundles()
                 self.wait_for_analysis_workflows()
                 self.wait_for_secondary_bundles()
-            if self.dataset.name == "Smart-seq2":
+            if self.dataset.name == "Smart-seq2" or self.dataset.name == "optimus":
```

Comment:
Not sure if we want to disable matrix for optimus too, @parthshahva ?

Reply:
I just noticed it's commented out on the `integration` branch so I will do the same here

Reply:
FYI: https://humancellatlas.slack.com/archives/G9XD6L0AD/p1553277875269900